scripts/colorTools.js

```diff
@@ -1,5 +1,13 @@
 "use strict"
 
+function createRBG(red, green, blue) {
```

Comment:
the function's name createRBG is incorrect. You should rename to createRGB (red, green blue).